Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_smorci/src/main.rs

```diff
@@ -0,0 +1,64 @@
+use num_bigint::{BigUint, ParseBigIntError};
+use std::io;
+use std::str::FromStr;
+
+const NUMBER_OF_DAYS_IN_A_YEAR: u32 = 365;
+
+fn convert_years_to_days(years: BigUint) -> BigUint {
```

Comment:
maybe this should be in a separate module, pending discussion on slack

Reply:
we decided to move it to a separate module. since it is the only module in the crate, it can be the top-level module (`lib.rs`)

Reply:
:+1: